docs/ref/contrib/csp/configuration.rst

```diff
@@ -0,0 +1,85 @@
+.. _configuration-chapter:
+
+===============
+Configuring csp
+===============
+
+Content-Security-Policy_ is a complicated header. There are many values
+you may need to tweak here.
+
+.. note::
+   Note when a setting requires a tuple or list. Since Python strings
+   are iterable, you may get very strange policies and errors.
```

Comment:
Django detects and automatically fixes this sort of sloppiness is various places. Automatically fixing or at least providing a clear error in such cases is better than warning "hey you'll get confusing errors".